app/components/works/add_files_component.html.erb

```diff
@@ -6,7 +6,7 @@
                               bs_toggle: 'modal',
                               bs_target: '#contactUsModal' }) %>
   </p>
-  <div class="dropzone dropzone-default<%= ' is-invalid' if error? %>" data-dropzone-target="container">
+  <div class="dropzone dropzone-default" data-dropzone-target="container">
```

Comment:
It seemed that this div was not included in what `validate-forms.js` checks when looking for 'is-invalid' on form elements.